File: CodeChef/Ranking1000to1200/SlowSolution/mySolution.java
import java.util.Scanner;
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner input = new Scanner(System.in);
		int t, maxt, maxn, sumn, a, b,n;
		t=input.nextInt();
		for(t=t; t>0; t--)
		{
		    maxn=input.nextInt();
		    maxt=input.nextInt();
		    sumn=input.nextInt();
		    if(sumn%maxt>0)
		    {
		        n=sumn/maxt;
		        a=maxt;
		        if(n>=maxn)
		        {
		            n=maxn;
		            b=0;
		        }else{
		            b=sumn%maxt;
		        }
		    }
		    else
		    {
		        n=sumn/maxt;
		        b=0;
		        a=maxt;
		        if(n>maxn)
		        {
		            n=maxn;
		        }
    		}
    		System.out.println((n*((int)Math.pow(a,2)))+((int)Math.pow(b,2)));
		}
	}
}

File: CodeChef/Ranking1000to1200/ChefandNumberGame/mySolution.py
t=int(input())
while t>0:
    t-=1 
    n=int(input())
    v=list(map(int, input().split(' ')))
    s1=0
    s2=0
    t1=0
    t2=0
    for i in v:
        if i>0:
         t1+=1
        else:
         t2+=1
    if t1==0:
        print(t2, t2)
    elif t2==0:
        print(t1, t1)
    else:
        print(t1, t2)
